packages/renderer/src/lib/dialogs/dialog-utils.ts

```diff
@@ -0,0 +1,34 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+export class DialogUtils {
```

Comment:
I'm wondering why you didn't directly exported a function

```
export function tabWithinParent(...
```

as the class is not required here as you don't want to instantiate anyway. Probably a Java habit.

Reply:
lol, yes, old habits die hard. New commit fixes this.

Reply:
it was just a remark. thanks for the refactoring 👍